traitsui/__init__.py

```diff
@@ -27,7 +27,14 @@
     "pyqt": ["pyqt>=4.10", "pygments"],
     "pyqt5": ["pyqt5", "pygments"],
     "pyside2": ["pyside2", "shiboken2", "pygments"],
-    "demo": ["configobj", "docutils"],
+    "examples": [
```

Comment:
It was called `demo` because it was the dependencies for the *demo application*.
To be fair, the `traitsui.extras.demo` is still here. But I think we can get rid of that pretty soon now that `etsdemo` is released.

Reply:
Oops, I thought it was added recently, but it isn't: https://github.com/enthought/enable/commit/c6e7664d8e91180a17eaaf9ce60dfd5218712eb5
I have to add the `demo` back...